app/src/main/java/com/gwu/android/androidtweets/TweetsActivity.kt

```diff
@@ -22,7 +25,27 @@ class TweetsActivity : AppCompatActivity() {
 
         recyclerView = findViewById(R.id.recyclerView)
         recyclerView.layoutManager = LinearLayoutManager(this)
-        // recyclerView.adapter = TODO next week
+
+        recyclerView.adapter = TweetsAdapter(getFakeTweets(), this)
+    }
+
+    override fun onRowItemClicked(tweet: Tweet) {
```

Comment:
Note -- the implementation of this function was just for demonstrating dialogs & click listeners.

Our Android Tweets app doesn't need any practical functionality (yet) when you click on a particular row.